numpy/core/src/umath/override.c

```diff
@@ -45,9 +45,9 @@ normalize___call___args(PyUFuncObject *ufunc, PyObject *args,
      */
     int i;
     int not_all_none;
-    int nin = ufunc->nin;
-    int nout = ufunc->nout;
-    int nargs = PyTuple_GET_SIZE(args);
+    npy_intp nin = ufunc->nin;
+    npy_intp nout = ufunc->nout;
+    npy_intp nargs = PyTuple_GET_SIZE(args);
```

Comment:
Can we leave fixing these remaining warnings until that time then? With this change, I get fewer warnings on MSVC, which insists on only showing warnings when an error is elsewhere